doc/manual/source/rtr-service.rst

```diff
@@ -2,48 +2,76 @@ RTR Service
+In order to start the RTR server at 192.0.2.13 and 2001:0DB8::13 on port
+3323, run Routinator using the :subcmd:`server` subcommand:
 
 .. code-block:: text
 
    routinator server --rtr 192.0.2.13:3323 --rtr [2001:0DB8::13]:3323
 
 Please note that port 3323 is not the :abbr:`IANA (Internet Assigned Numbers
-Authority)`-assigned default port for the protocol,  which would be 323. But as
-this is a privileged port, you would need to be running Routinator as root when
-otherwise there is no reason to do that. 
+Authority)`-assigned default port for the protocol, which would be 323. But
+as this is a privileged port, you would need to be running Routinator as root
+when otherwise there is no reason to do that. 
 
-Communication between Routinator and the router using the RPKI-RTR protocol is
-done via plain TCP. Below, there is an explanation how to secure the transport
-using either SSH or TLS.
+Secure Transport
+----------------
 
-Secure Transports
------------------
+.. versionadded:: 0.11.0
+   RTR-over-TLS connections 
 
-These instructions were contributed by `wk on Github
-<https://github.com/NLnetLabs/routinator/blob/master/doc/transports.md>`_.
+Although there is no mandatory-to-implement transport that provides
+authentication and integrity protection, :rfc:`6810#section-7` defines a
+number of secure transports for RPKI-RTR that can be used to secure
+communications. This includes TLS, SSH, TCP MD5 and TCP-AO transports. 
 
-:rfc:`6810#section-7` defines a number of secure transports for RPKI-RTR that
-can be used to secure communication between a router and a RPKI relying party.
+Routinator currently has native support for TLS connections, and can be
+configured to use `SSH`_ with some additional tooling.
 
-However, the RPKI Router Implementation Report documented in
-:rfc:`7128#section-5` suggests these secure transports have not been widely
-implemented. Implementations, however, do exist, and a secure transport could be
-valuable in situations where the RPKI relying party is provided as a public
-service, or across a non-trusted network.
+TLS
+"""
 
-SSH Transport
-"""""""""""""
+It's possible to natively use RTR-over-TLS connections with Routinator. The
+requirements are described in detail in :rfc:`6810#section-7.2`. There is an
+:abbr:`IANA (Internet Assigned Numbers Authority)`-assigned default port for
+rpki-rtr-tls as well, in this case 324.
+
+Currently, very few routers have implemented support for TLS, but it may be
+especially useful to use secure connections when deploying our RTR data proxy
+:doc:`RTRTR <rtrtr:index>`, as data may be flowing across the public
+Internet.
+
+In this example we'll start Routinator's RTR server listening on the IP
+addresses 192.0.2.13 and 2001:0DB8::13 and use port 3324 to make sure it's
+not a priviledged port. Use the :option:`--rtr-tls` to indicate that this is
+a TLS connection and the :option:`--rtr-tls-cert` option to specify the path
```

Comment:
You also need `--rtr-tls-key` to specify the private key to be used. Other than stated below, this is mandatory – we need to private key separately since the certificate only includes the public key.